Manifests/ManagedPreferencesApplications/com.github.mpanighetti.install-or-defer.plist

```diff
@@ -0,0 +1,410 @@
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Payload Version</string>
+			<key>pfm_type</key>
+			<string>integer</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>This value describes the issuing organization of the profile, as displayed to the user</string>
+			<key>pfm_name</key>
+			<string>PayloadOrganization</string>
+			<key>pfm_title</key>
+			<string>Payload Organization</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>5.0</string>
+			<key>pfm_default</key>
+			<string>Install</string>
+			<key>pfm_description</key>
+			<string>The label of the install button.</string>
+			<key>pfm_description_reference</key>
+			<string>The label of the install button. Defaults to "Install". Keep this string short since jamfHelper will cut off longer button labels.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/mpanighetti/install-or-defer#configuration-profile</string>
+			<key>pfm_name</key>
+			<string>InstallButtonLabel</string>
+			<key>pfm_note</key>
+			<string>Keep this string short since jamfHelper will cut off longer button labels.</string>
+			<key>pfm_title</key>
+			<string>Install Button Label</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>Install</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>5.0</string>
+			<key>pfm_default</key>
+			<string>Defer</string>
+			<key>pfm_description</key>
+			<string>The label of the defer button.</string>
+			<key>pfm_description_reference</key>
+			<string>The label of the defer button. Defaults to "Defer". Keep this string short since jamfHelper will cut off longer button labels.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/mpanighetti/install-or-defer#configuration-profile</string>
+			<key>pfm_name</key>
+			<string>DeferButtonLabel</string>
+			<key>pfm_note</key>
+			<string>Keep this string short since jamfHelper will cut off longer button labels.</string>
+			<key>pfm_title</key>
+			<string>Defer Button Label</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>Defer</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>5.0</string>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Whether to write to a persistent log file at /var/log/install-or-defer.log.</string>
+			<key>pfm_description_reference</key>
+			<string>Whether to write to a persistent log file at /var/log/install-or-defer.log. Defaults to false, instead writing all output to the system log for live diagnostics.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/mpanighetti/install-or-defer#configuration-profile</string>
+			<key>pfm_name</key>
+			<string>DiagnosticLog</string>
+			<key>pfm_title</key>
+			<string>Diagnostic Log</string>
```

Comment:
Can this be reworded into _Persist Diagnostic Log_? There's no description for booleans in IPE so this would give a bit more context.

Reply:
Renamed to "Enable Diagnostic Log" to better indicate it's a boolean key.